engine/compiler/compiler.go

```diff
@@ -427,6 +437,35 @@ func (c *Compiler) Compile(ctx context.Context, args runtime.CompilerArgs) runti
 		}
 	}
 
+	// create internal steps if build running in debug mode
+	if c.Tmate.Enabled && args.Build.Debug && pipeline.Platform.OS != "windows" {
+		// first we need to add an internal setup step to the pipeline
+		// to copy over the tmate binary. Internal steps are not visible
+		// to the end user.
+		spec.Internal = append(spec.Internal, &engine.Step{
+			ID:         random(),
+			Pull:       engine.PullIfNotExists,
+			Image:      image.Expand(c.Tmate.Image),
+			Entrypoint: []string{"/bin/sh", "-c"},
```

Comment:
let's use the native `drone cp` command that is built-in to the runner.  The reason is the arm32 and arm64 images are scratch images and do not have the `cp` command available.

Example at: https://github.com/drone-runners/drone-runner-docker/blob/master/engine/compiler/compiler.go#L350:L351

Reply:
Done